pkg/action/action_test.go

```diff
@@ -44,3 +43,47 @@ func TestMakeLogWriter(t *testing.T) {
 		t.Errorf("error deleting file; file: [%s], error: [%s]", logName, err)
 	}
 }
+
+func TestLazyFile(t *testing.T) {
+	testFileNamePrefix := filepath.Join(os.TempDir(), "TestLazyFile")
+	testFileContent := []byte("test lazy file content")
+
+	lzFile := &lazyFile{out: nil, name: "", prefix: testFileNamePrefix}
```

Comment:
doesn't this (shorter) version work?
```suggestion
	lzFile := &lazyFile{prefix: testFileNamePrefix}
```